spikeinterface/widgets/sorting_summary.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+
+from .base import BaseWidget, define_widget_function_from_class
+from .widget_list import AutoCorrelogramsWidget, CrossCorrelogramsWidget, UnitWaveformsWidget, AmplitudeTimeseriesWidget
+from ..core import WaveformExtractor
+from ..postprocessing import get_template_channel_sparsity, compute_template_similarity
+
+
+class SortingSummaryWidget(BaseWidget):
+    """
+    Plots spike sorting summary
+    
+    Parameters
+    ----------
+    waveform_extractor: WaveformExtractor
+    """
+    possible_backends = {}
+
+    
+    def __init__(self, waveform_extractor: WaveformExtractor, unit_ids=None,
+                 sparsity=None, sparsity_kwargs={}, 
+                 correlograms_kwargs=None, amplitudes_kwargs=None, 
+                 similarity_kwargs=None, job_kwargs=None,
+                 # compute_pca_kwargs=None, localization_kwargs=None,
+                 backend=None, **backend_kwargs):
+        self.check_backend(backend)
```

Comment:
This should be done in base widget no ? This would avoid to copy/paste anyware and face a bug we forgit it.